tests/MongoDB.Driver.Tests/Specifications/unified-test-format/UnifiedEntityMap.cs

```diff
@@ -0,0 +1,484 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using MongoDB.Bson;
+using MongoDB.Driver.Core;
+using MongoDB.Driver.Core.Events;
+using MongoDB.Driver.GridFS;
+using MongoDB.Driver.TestHelpers;
+
+namespace MongoDB.Driver.Tests.Specifications.unified_test_format
```

Comment:
After moving this file to the `UnifiedTestOperations` directory change the namespace to:

    namespace MongoDB.Driver.Tests.UnifiedTestOperations